src/calculateRentalCost.js

```diff
@@ -5,6 +5,23 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const CONST_ONE_DAY = 40;
+  const RENT_FOR_7_DAYS = 7;
+  const RENT_FOR_3_DAYS = 3;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  if (days >= RENT_FOR_7_DAYS) {
+    return CONST_ONE_DAY * days - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= RENT_FOR_3_DAYS && days <= 6) {
```

Comment:
Don't use literals in expressions (HARDCODE). Instead use constants explaining this values.

![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/54860641/5e928e82-7df1-4f0f-8c7c-a67778c3ec47)